src/Utils/Helpers.ts

```diff
@@ -64,16 +66,19 @@ export class Helpers {
       if (['localhost', '127.0.0.1'].includes(host)) {
         bus = `amqp://${host}:5672`;
         api = `http://${host}:8000`;
+        repeaterServer = `wss://${host}/workstations`;
```

Comment:
For local development, we typically utilize the non-secure 8000 port.





```suggestion
        repeaterServer = `ws://${host}:8000/workstations`;
```